proto/c1/connector/v2/action.proto

```
@@ -0,0 +1,79 @@
+
+package c1.connector.v2;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/struct.proto";
+import "c1/config/v1/config.proto";
+
+option go_package = "github.com/conductorone/baton-sdk/pb/c1/connector/v2";
+
+
+service ActionService {
+  rpc InvokeAction(ActionServiceInvokeRequest) returns (ActionServiceInvokeResponse);
```

Buf (1.47.2)

RPC request type "ActionServiceInvokeRequest" should be named "InvokeActionRequest" or "ActionServiceInvokeActionRequest".

(RPC_REQUEST_STANDARD_NAME)